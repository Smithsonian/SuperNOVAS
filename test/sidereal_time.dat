
2450545.000000 TEST       S2 O0 A1:     0.987551     0.987551

2450545.000000 TEST       S2 O1 A1:     0.987551     0.987551

2450545.000000 TEST       S2 O2 A1:     0.987551     0.987551

2450545.000000 SUN        S0 O0 A1:     0.987551     0.987551

2450545.000000 SUN        S0 O1 A1:     0.987551     0.987551

2450545.000000 SUN        S0 O2 A1:     0.987551     0.987551

2451545.000000 TEST       S2 O0 A1:    18.697375    18.697375

2451545.000000 TEST       S2 O1 A1:    18.697375    18.697375

2451545.000000 TEST       S2 O2 A1:    18.697375    18.697375

2451545.000000 SUN        S0 O0 A1:    18.697375    18.697375

2451545.000000 SUN        S0 O1 A1:    18.697375    18.697375

2451545.000000 SUN        S0 O2 A1:    18.697375    18.697375

2452545.000000 TEST       S2 O0 A1:    12.407199    12.407199

2452545.000000 TEST       S2 O1 A1:    12.407199    12.407199

2452545.000000 TEST       S2 O2 A1:    12.407199    12.407199

2452545.000000 SUN        S0 O0 A1:    12.407199    12.407199

2452545.000000 SUN        S0 O1 A1:    12.407199    12.407199

2452545.000000 SUN        S0 O2 A1:    12.407199    12.407199
