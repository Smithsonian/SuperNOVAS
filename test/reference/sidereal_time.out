-10000.0 A0:     9.599134     9.599134
     0.0 A0:    18.697375    18.697375
 10000.0 A0:     3.795620     3.795620
-10000.0 A1:     9.599134     9.599134
     0.0 A1:    18.697375    18.697375
 10000.0 A1:     3.795620     3.795620
